### ISSUE TITLE
MO2 Not properly addressing Meshes, which generally leads to crashes

### ISSUE THREAD

**Issue by FuHeng69** [TanninOne/modorganizer#435]
Hi guys, I was able to do some research on an issue I was having.

1) I downloaded http://www.nexusmods.com/fallout4/mods/1478/? , Homemaker and when I tried to use the institute stuff it lead to a CTD. After rigorous investigation I concluded that there had to be an issue with the meshes.

2) To confirm #1 I went and downloaded http://www.nexusmods.com/fallout4/mods/2197/? , Upon load I added the item through console and noticed that the meshes did not load at all, leading to my character's full body becoming invisible.

3) I tried messing around with the INI files both through MO, and also the ones located in the documents. I made sure archive invalidation was properly adjusted bInvalidateOlderFiles=1 and tried with and without the strings in sResourceDataDirsFinal

4) #3 did not work, so I went ahead and made sure the problem was isolated to MO, I installed both mods successfully through NMM. Leading to the conclusion that the current alpha version of MO cannot properly handle meshes.

5) hope this information useful in your next release.

**Comment by berserkhealer** [comment]
Indeed, this is the case. It appears that MO2a2 isn't properly handling meshes, at least on Windows 8.1 x64. 

I've been using the Armorsmith Extended mod and decided to add the Wearable Squire Uniform mod, which adds some new standalone meshes for the backpack/hat/uniform separately.

If I install the Wearable Squire Uniform mod through MO2, the standalone meshes, such as the backpack, do not appear. However, manual installation allows the items to appear. So it appears this is not an isolated issue and that MO2 can't process the virtual meshes directory properly. 

Hope this gets fixed next release. Keep up the great work, Tannin -- I've been using MO for my FNV and Skyrim installs and look forward to using MO2 for FO4 for the long term!
